draft-normington-jsonpath-latest.txt

```diff
@@ -424,20 +424,20 @@ Internet-Draft   JavaScript Object Notation (JSON) Path        July 2020
    selects the value corresponding to the key.  It does not match a node
    which is not an object.
 
-2.6.2.4.  Array Index
+2.6.2.4.  Array Access
 
 Syntax
 
+   Array access is a union element which matches any node which is an
+   array and which selects zero or more elements of the array.  Array
```

Comment:
I found this difficult to follow. Perhaps something like "Array access is a union element which matches zero or more elements of an array node."

Reply:
I'm using the terminology of a matcher _matching_ a node and then _selecting_ subnodes. There's a clear distinction for example with "dot child" where the key matches the key in an object but the matcher selects the value subnode. In the array case, we can probably get away with shortening this. Let's see how that pans out.